rust/benchmarks/src/bin/tpch.rs

```diff
@@ -1677,7 +1677,6 @@ mod tests {
             let actual_vec = result_vec(&actual);
 
             // basic result comparison
-            assert_eq!(expected_vec.len(), actual_vec.len());
```

Comment:
Just curious: Is there a reason to remove this? The next `for` loop does not check the results' lens (i.e. if `actual_vec.len() < expected_vec.len()` and has equal rows, then the check passes).

Reply:
Will have a look later at this. I thought it was failing as it returns a vec of batches(?) but not sure now.

Reply:
Probably have to flatten the `Vec`  to make the test pass.

Reply:
I reverted this change -> found out the test was actually wrong..

Also interesting, I thought one of the tests was giving a wrong example (besides some rounding errors because we don't use decimals in the benchmark), but it seems currently wrong on master too:

```
Query 6 iteration 0 took 6137.1 ms
Query 6 avg time: 6137.09 ms
thread 'tests::q6' panicked at 'assertion failed: `(left == right)`
  left: `["123141078.23"]`,
 right: `["75207768.18550001"]`', benchmarks/src/bin/tpch.rs:1684:17
```

Reply:
is there a JIRA ticket tracking this? I would perhaps be interested in looking at it

Reply:
Created :+1: